debug/scroll_zoom_blocker.html

```diff
@@ -41,14 +41,25 @@
             margin-left: 18px;
             font-size: 12px;
         }
-
+        #globe { 
+            position: absolute;
+            top: 0px;
+            left: 370px;
+            margin: 15px;
+            font-size: 14px;
+            padding: 8px;
+            background-color: lightskyblue;
+            color: darkblue;
+            font-weight: bold;
+        }
     </style>
 </head>
 
 <body>    
 <div id="map-container">
     <div id="map"></div>
 </div>
+<button id="globe">Switch to globe</button>
```

Comment:
Have you tested how this appears on mobile? Would it be possible to the UI more compact by putting the button inside of the div element below?

Reply:
Yeah I've always thought the UI on this page could use improvement! I look about putting the button under the map but thought it would be harder to miss if it was by the checklist

Reply:
I agree that under the map is probably not the best place for it, my thought was that putting it inside of the checkboxes div could make the UI take up less space. Or CSS changes could reduced the white space.

Another option could be putting it over the map like in other examples. (I think using `position: absolute`?)

Reply:
Ok sounds good, since it is combining canvas size - the small size with globe makes a not great UI experience